esp/services/ws_workunits/ws_workunitsHelpers.cpp

```diff
@@ -3814,7 +3816,7 @@ void CWsWuFileHelper::createWUZAPFile(IEspContext& context, IConstWorkUnit* cwu,
     StringBuffer& zipFileName, StringBuffer& zipFileNameWithPath, unsigned _thorSlaveLogThreadPoolSize)
 {
     StringBuffer zapReportNameStr, folderToZIP, inFileNamePrefixWithPath;
-    Owned<IFile> zipDir = createWorkingFolder(context, request.wuid.str(), "ZAPReport_", zapReportNameStr, folderToZIP);
+    Owned<IFile> zipDir = createWorkingFolder(context, request.wuid.str(), "ZAPReport_", false, zapReportNameStr, folderToZIP);
```

Comment:
@jakesmith any idea why I got "Unable to create temp directory"?

Reply:
@wangkx  have you checked to see if the parent directory exists before mkdtemp is called?
I think it will need to.
If so, may need a recursiveCreateDirectory(dir) after the getConfigurationDirectory call.